locales/es/LC_MESSAGES/volto.po

```diff
@@ -2606,7 +2607,7 @@ msgstr "Enviar"
 #: helpers/MessageLabels/MessageLabels
 # defaultMessage: Send a confirmation mail with a link to set the password.
 msgid "Send a confirmation mail with a link to set the password."
-msgstr ""
+msgstr "Envíe un correo de confirmación con un enlace para establecer la contraseña."
```

Comment:
Envíe vs. Enviar? have to check where this is used

Reply:
Done with commit 3209ec4